### DISCUSSION TITLE
result = inference_detector()  result do not show bboxes

### DISCUSSION THREAD

**Discussion by zhaohai3424** [open-mmlab/mmdetection#9313] [Q&A] [answered]
### Discussed in https://github.com/open-mmlab/mmdetection/discussions/9278

<div type='discussions-op-text'>

<sup>Originally posted by **zhaohai3424** November  8, 2022</sup>
I USE faster-rcnn_r50_fpn_1x_COCO.py  train my dataset successfully.

i use inference.py to test my picture,and many pictrues show the right detection,howerver some did not show any result, as below,pic1 is right.pic2 is wrong
![图片](https://user-images.githubusercontent.com/117725731/200536786-95a1cffb-123c-40a0-901a-dc5b714497d3.png)

![图片](https://user-images.githubusercontent.com/117725731/200536936-608c7e6f-fcf7-4bed-8af9-5074c7e91d18.png)
do you guys known the reason?i will appriciate it,thanks
</div>

**Comment by zhaohai3424** [comment, marked as answer]
i find the answer,and share it with you ,in case you meet the same questions;

![图片](https://user-images.githubusercontent.com/117725731/200718553-c3a527c6-bd9b-400e-95e9-2d7f9580dbf7.png)

**Comment by ZwwWayne** [comment]
This is because there is no boxes predicted aftering filtering by the score threshold. Therefore, you can try to make it smaller.
